tests/fixtures/code_objects/blocksworld_fond.py

```diff
@@ -60,7 +60,7 @@ def blocksworld_fond_domain():
         ),
         AndEffect(
             on(x, table),
-            When(~EqualTo(z, table), ~on(x, z) & clear(z)),
```

Comment:
This is probably the biggest impact of this change...using the shorthand `&` between fluents will automatically use the `And`, but if it's an effect, then it should be an `AndEffect` (I think?)